src/components/KeplerGL/index.js

```diff
@@ -65,7 +65,10 @@ function getHoverInfo(info, allData) {
     : // if hovered is a hexbbin, or grid, kepler.gl can return all the points inside that hexagon / grid
     objectHovered.type === 'Feature'
     ? allData[objectHovered.properties.index]
-    : objectHovered.points;
+    // with upgrade to v2 we now need to the data sub-object for hex and grid layer
+    : objectHovered.points.map(pt => {
```

Comment:
there should be a `filteredPoints` property, that the filtered data

Reply:
I am going to separate out the enhancement for new selection tool/filtering via this issue/request https://github.com/keplergl/kepler.gl-tableau/issues/22. This PR will be scoped to only fixing the broken functionality around integration with points and/or aggregated marks (e.g., hex layer).